docs/openapi/ms-auth.yaml

```diff
@@ -134,12 +132,11 @@ paths:
                 example: jwt_token=; Max-Age=0; HttpOnly
           content:
             application/json:
-              example: {"message": "Successful Logout (Token Deleted)"}
-        '404':
-          description: Token does not exist
-          content:
-            application/json:
-              example: {"message": "Token does not exist"}
+              examples:
+                user_logged_out:
+                  value: {"message": "User successfully logged out"}
+                user_not_logged_in:
+                  value: {"message": "User was not logged in"}
```

Comment:
@cryptoclown132 i changed 404 to 200 if a user was not logged in, because it was confusing to get an error in the console